scanner/decisions/0003-scanner-image-names.md

```diff
@@ -0,0 +1,63 @@
+# 0003 - Scanner image names
+
+- **Author(s):** Ross Tannenbaum <rtannenb@redhat.com>
+- **Created:** 2023-08-22
+
+## Status
+
+Accepted.
+
+## Context
+
+The [current version of StackRox Scanner](https://github.com/stackrox/scanner) shipped with each StackRox release is a fork of [Clair v2](https://github.com/quay/clair/tree/v2.1.8) which is versioned and released separately from the rest of StackRox.
+The separate versioning process has proven to cause confusion in the past, so we re-tag each Scanner release with the associated StackRox release.
+For example, the Scanner 2.30.3 release triggered the creation of the following images:
+
+* scanner:2.30.3
+* scanner-slim:2.30.3
+* scanner-db:2.30.3
+* scanner-db-slim:2.30.3
+
+Upon cutting the related StackRox release, 4.1.3, the images were re-tagged as follows:
+
+* `scanner:4.1.3`
+* `scanner-slim:4.1.3`
+* `scanner-db:4.1.3`
+* `scanner-db-slim:4.1.3`
+
+There is an ongoing effort to overhaul the StackRox Scanner to align it more closely with [Clair v4](https://github.com/quay/clair).
```

Comment:
Is it correct that Clair is not bumped from Clair v4 to Clair v5 as part of this effort?

Reply:
Correct. Clair is currently on v4 with no plans to bump to a v5 any time soon (that I'm aware of)